packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/servers/server/dashboard/dashboard.html

```diff
@@ -0,0 +1,227 @@
+<div data-ui-view="dashboard">
+    <!-- Messages -->
+    <!-- Install in progress -->
+    <oui-message
+        class="mb-3"
+        data-type="info"
+        data-dismissable
+        data-ng-if="$ctrl.dedicatedServer.$scope.disable.installationInProgress && !$ctrl.dedicatedServer.$scope.disable.installationInProgressError"
+    >
+        <span data-translate="server_tab_dashboard_install_progress"> </span>
+        <button
+            type="button"
+            class="oui-button oui-button__link p-1"
+            data-translate="server_tab_dashboard_install_progress_link"
+            data-ui-sref="app.dedicated-server.server.dashboard.installation-progress"
+        ></button>
+    </oui-message>
+
+    <!-- Install error -->
+    <oui-message
+        class="mb-3"
+        data-type="error"
+        data-dismissable
+        data-ng-if="$ctrl.dedicatedServer.$scope.disable.installationInProgressError"
```

Comment:
**question:** Is it not possible to avoid usage of `$scope`?

Reply:
In fact, there's a triangle of information between dashboard, server-installation and server-installation-progress.
We need to check that the same information is not being used by the other 2 modules.